tests/baselines/reference/quickInfoDisplayPartsUsing.baseline

```diff
@@ -0,0 +1,174 @@
+=== /tests/cases/fourslash/quickInfoDisplayPartsUsing.ts ===
+// using a = "a";
+//       ^
+// | ----------------------------------------------------------------------
+// | using a: "a"
+// | ----------------------------------------------------------------------
+// const f = async () => {
+//     await using b = { async [Symbol.asyncDispose]() {} };
+//                 ^
+// | ----------------------------------------------------------------------
+// | await using b: {
```

Comment:
Do we actually want to distinguish `await` here? I feel like it'd be acceptable to just have `using b` but I'm unsure.

@DanielRosenwasser

Reply:
IMO the distinction is relevant because it affects whether it will call `dispose` vs. `asyncDispose` (i.e. the same reason to make the distinction between `using` and e.g. `const`—the runtime behavior differs)

Reply:
Agreed that the distinction is relevant. It's also just part of the declaration.